src/main/drivers/usb_msc_f4xx.c

```diff
@@ -145,11 +145,20 @@ void mscWaitForButton(void)
     }
 }
 
-void systemResetToMsc(void)
+void systemResetToMsc(int timezoneOffsetMinutes)
 {
     *((uint32_t *)0x2001FFF0) = MSC_MAGIC;
 
     __disable_irq();
+    
```

Comment:
Nitpicking here, but shouldn't this be done before diabling IRQs?

Reply:
Not sure it matters as we're doing a reboot a couple of instructions later. But I'll have other changes so I can revise.